README.md

```diff
@@ -83,12 +104,65 @@ the koji certificates directory (__~/.local/share/osbs-box/certificates/koji/__
 There is one for each koji user (by default, only _kojiadmin_ and _kojiosbs_ are users, but
 logging in creates a user automatically).
 
-#### Koji CLI (local)
+### Koji CLI (local)
 
 Coming soon<sup>TM</sup>
 
+### Docker registry
```

Comment:
`s/docker/container/`

Reply:
Do I `s/docker/container/` for every single instance where the registry is mentioned? Or just this title? For what it's worth, it _is_ a __docker__ registry.